extensions/interactions/EndExploration/directives/EndExplorationValidationService.js

```diff
@@ -21,10 +21,21 @@ oppia.factory('EndExplorationValidationService', [
   function(WARNING_TYPES, baseInteractionValidationService) {
     return {
       getCustomizationArgsWarnings: function(customizationArgs) {
+        var warningsList = [];
         baseInteractionValidationService.requireCustomizationArguments(
           customizationArgs, ['recommendedExplorationIds']);
-        // TODO(juansaba): Implement customization args validations.
-        return [];
+
+        var recommendedExplorationIds = (
+            customizationArgs.recommendedExplorationIds.value);
```

Comment:
Should you also check in the first place that the whole thing is a list?

Reply:
The value should be always a list as per the schema for recommendedExplorationIds. Should we need to explicitly check it?

Reply:
Yeah, might as well -- let's do full validation.

Reply:
Done, PTAL!